crates/next-custom-transforms/tests/fixture/server-actions/server-graph/58/output.js

```diff
@@ -2,10 +2,11 @@
 import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
 import { cache as $$cache__ } from "private-next-rsc-cache-wrapper";
 import { cache as $$reactCache__ } from "react";
+const $$RSC_SERVER_CACHE_0_INNER = async function([$$ACTION_ARG_0]) {
+    return $$ACTION_ARG_0();
+};
 export var $$RSC_SERVER_CACHE_0 = $$reactCache__(function() {
-    return $$cache__("default", "c03128060c414d59f8552e4788b846c0d2b7f74743", 1, async function([$$ACTION_ARG_0]) {
-        return $$ACTION_ARG_0();
-    }, arguments);
+    return $$cache__("default", "c03128060c414d59f8552e4788b846c0d2b7f74743", 1, $$RSC_SERVER_CACHE_0_INNER, arguments);
```

Comment:
Aren't these functions now leaking our wrapper names? Or is `$$RSC_SERVER_CACHE_0_INNER` not a userspace function?

Before this change the function was anonymous but now it's named `$$RSC_SERVER_CACHE_0_INNER`

Reply:
I see. Yeah, when a function does not have a name originally, I guess we would leak the internal names. Need to think about this case...

Reply:
I think that's a good solution for those edge cases. 👍 It might also be needed when building with `--debug-prerender`, for example. So I didn't only do it for dev in 00f8949.